loader/windows/icd_windows.c

```diff
@@ -423,7 +423,7 @@ void *khrIcdOsLibraryLoad(const char *libraryName)
     }
     if (!hTemp)
     {
-        KHR_ICD_TRACE("Failed to load driver. Windows error code is %d.\n", GetLastError());
+        KHR_ICD_TRACE("Failed to load driver. Windows error code is %lu.\n", GetLastError());
```

Comment:
I'm OK with this as-is and this is definitely an improvement, but a slightly more portable option would be to use `PRIuDW` here instead.  We used this a couple of other places in the OpenCL ICD loader to print `DWORD` values. 
 Example:
```
KHR_ICD_TRACE("Reading value %"PRIuDW"...\n", dwIndex);
```

Reply:
Fixed this. 👍